.github/workflows/generate_clients.yaml

```diff
@@ -0,0 +1,55 @@
+name: Generate clients
+
+on:
+  workflow_dispatch:
+
+jobs:
+  generate:
+    name: Generate Clients
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: add ssh key for use in deps installation
+        run: |
+          # this link below looks like it will mostly impact executioner, but that is a dep of cloud-api via client
+          # needed for libcurl - https://github.com/aptible/executioner-clients/runs/7506231404?check_suite_focus=true
+          #  FileNotFoundError: [Errno 2] No such file or directory: 'curl-config'
+          sudo apt-get update && sudo apt-get install -yq libcurl4-openssl-dev libssl-dev
+          mkdir -p ~/.ssh && \
+            ssh-keyscan -t rsa github.com >> ~/.ssh/known_hosts
+
+        # ssh key is for building
+      - uses: webfactory/ssh-agent@v0.5.4
+        with:
+          ssh-private-key: ${{ secrets.APTIBLE_ROBOT_PRIVATE_SSH_KEY }}
+
+      # 1- Install dependencies to setup the repo
+      - uses: actions/setup-python@v3
+        with:
+          python-version: '3.10'
+
+      - name: Install dependencies
+        run: |
+          make reclone
+          pip install -e .[dev]
+
+      - name: generate openapi.json for reuse
+        run: |
+          source ./scripts/setenvs.sh
+          python -m scripts.openapi.generate openapi.json
+        env:
+          BUNDLE_DIRECTORY: "/home/runner/work/cloud-api-clients/cloud-api-clients/bundles"
+
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 16.x
```

Comment:
In general I recommend putting all of the setup actions at the top together, so if they fail it fails quickly.

Reply:
i think it gets pulled to the top but ill do that, good point

Reply:
it does not get pulled to the top

<img width="921" alt="image" src="https://user-images.githubusercontent.com/2961973/181059844-866e74f5-e998-48e1-b459-51b3900ba40e.png">